agent/src/agent/pipeline/validators/jdbc_query.py

```diff
@@ -1,8 +1,12 @@
-from agent import pipeline
+from agent.pipeline.jdbc.query import TIMESTAMP_CONDITION, TIMESTAMP_COLUMN, LAST_TIMESTAMP_VALUE, INTERVAL
 
 
 def get_errors(query: str):
     errors = []
-    if pipeline.jdbc.query.TIMESTAMP_CONDITION not in query:
-        errors.append(f'Please add {pipeline.jdbc.query.TIMESTAMP_CONDITION} constant to the query')
+    if TIMESTAMP_CONDITION not in query:
+        errors.append(f'Please add {TIMESTAMP_CONDITION} constant to the query')
```

Comment:
if there's no TIMESTAMP_CONDITION but other fields are present, the validation will fail, but as I understand it should fail only if there are no fields at all

Reply:
There is errors.pop() in else statement for inner 'if'
So, if there are no TIMESTAMP_CONDITION and no [TIMESTAMP_COLUMN, LAST_TIMESTAMP_VALUE, INTERVAL] -> we will have two errors.
But if no TIMESTAMP_CONDITION and else present 'NO-TIMESTAMP' error will be popped

Reply:
just don't put the error there if you don't need it

Reply:
I mean change the code so that errors are added only in cases where there certainly is an error